### DISCUSSION TITLE
What's the state of inertia?

### DISCUSSION THREAD

**Discussion by francoism90** [inertiajs/inertia#1986] [Help] [answered]
I'm confused after reading the following Reddit post: https://www.reddit.com/r/laravel/comments/1bhsme2/what_is_the_actual_state_of_inertiajs/

I would like to start a new project myself with Inertia.js + VueJS 3. When checking out the PRs being merged, this project seems to be very active, and an update seems to be planned for next week?

On Reddit it's referred Inertia.js as 'completed software', but is this actually true? Is there any roadmap or ideas/features-planned available?

I understand this being more a bridge between front-end (VueJS, React and Svelte), it's actually purpose is to keep them integrated with the Laravel API, and not to provide load of features.

Could you please tell me what to expect in the near future? :)

Thanks for maintaining this, I just wanted to have developer view about this.

**Comment by marcosarce** [comment, marked as answer]
New features for Inertia 2.0 from Laracon US: https://youtu.be/AwWepVU5uWM?si=pX0uW_UlSX4m6_35&t=2927

**Reply by francoism90** [reply]
Thanks, that's all I needed to know. :)
